readme.md

```diff
@@ -55,6 +55,12 @@ Type: `boolean`
 
 Only show the first unit: `1h 10m` ➔ `~1h`.
 
+##### verbose
+
+Type: `boolean`
```

Comment:
I have added the `Default` option to both `verbose` and `compact`, it was missing earlier!